package.json

```diff
@@ -41,6 +42,7 @@
     "eslint": "^8.0.1",
     "eslint-config-prettier": "^8.3.0",
```

Comment:
I think you can move eslint, jest  and prettier to devDependencies, what do you think?

Reply:
Yeah, but eslint, jest and prettier are already in the devDependencies

Reply:
great!